lib/tasks/data_migration.rake

```diff
@@ -79,4 +79,62 @@ namespace :data_migration do
     puts "Associating protocol for #{organization.facility_groups.count} facility groups"
     organization.facility_groups.update(protocol: protocol)
   end
+
+  desc "Create automatic appointment for defaulters so that they show up in the Overdue List"
+  task create_automatic_appointment_for_defaulters: :environment do
+    last_bp_older_than_one_month = ->(p) { p.latest_blood_pressure.blank? || p.latest_blood_pressure.device_created_at < 1.month.ago }
+
+    patient_is_hypertensive = ->(p) { p.latest_blood_pressure.present? && (p.latest_blood_pressure.systolic > 140 || p.latest_blood_pressure.diastolic < 90) }
+
+    patient_has_medical_history = ->(p) { (p.medical_history.present? && (p.medical_history.prior_stroke == "yes" || p.medical_history.prior_heart_attack == "yes" || p.medical_history.chronic_kidney_disease == "yes" ||
```

Comment:
```suggestion
    patient_has_cardio_vascular_history = ->(p) { (p.medical_history.present? && (p.medical_history.prior_stroke == "yes" || p.medical_history.prior_heart_attack == "yes" || p.medical_history.chronic_kidney_disease == "yes" ||
```

Reply:
Use `#indicates_risk` from MedicalHistory

Reply:
Changed to use `indicates_risk?`